Doc/library/pdb.rst

```diff
@@ -241,10 +241,11 @@ middle of a quoted string.
    triple: debugger; configuration; file
 
 If a file :file:`.pdbrc` exists in the user's home directory or in the current
-directory, it is read in and executed as if it had been typed at the debugger
-prompt.  This is particularly useful for aliases.  If both files exist, the one
-in the home directory is read first and aliases defined there can be overridden
-by the local file.
+directory, it is read with ``'utf-8'`` encoding rather than system locale
```

Comment:
"rather than system locale encoding" is not needed here. It will rather confuse users. But please add the `versionchanged` directive in which describe the change.